homeassistant/components/hvv_departures/config_flow.py

```diff
@@ -114,9 +115,7 @@ async def async_step_station(self, user_input=None):
     async def async_step_station_select(self, user_input=None):
         """Handle the step where the user inputs his/her station."""
 
-        schema = vol.Schema(
-            {vol.Required(CONF_STATION): vol.In(list(self.stations.keys()))}
-        )
+        schema = vol.Schema({vol.Required(CONF_STATION): vol.In(list(self.stations))})
```

Comment:
We can remove `list` too as seen in my test.

Reply:
I had to undo that change, because otherwise this happens:

![image](https://user-images.githubusercontent.com/1506970/84538900-bf505f00-acf2-11ea-9072-be451be7b750.png)

Only if I add the list() around it, it works as intended:

![image](https://user-images.githubusercontent.com/1506970/84539058-0ccccc00-acf3-11ea-9807-26b8d584735a.png)

Reply:
Right, that's a feature of the frontend that you can use a dictionary and have it render the values in the list for the user form while using the keys as return values.